manifests/init.pp

```diff
@@ -6,8 +6,8 @@
 #  class { 'satellite_pe_tools':
 #    satellite_url => 'https://satellite.example.domain',
 #    ssl_ca        => '/etc/puppetlabs/puppet/ssl/ca/katello-default-ca.crt',
-#    ssl_cert      => '/etc/puppetlabs/puppet/ssl/certs/satellite-master.example.domain.pem',
-#    ssl_key       => '/etc/puppetlabs/puppet/ssl/private_keys/master.domain.com.pem',
+#    ssl_cert      => '/etc/puppetlabs/puppet/ssl/certs/satellite-server.example.domain.pem',
+#    ssl_key       => '/etc/puppetlabs/puppet/ssl/private_keys/puppet.domain.com.pem',
```

Comment:
Looking at https://github.com/puppetlabs/puppetlabs-satellite_pe_tools/blob/edf25e0e6f6ff5599dd4350690daa7dcb5e03647/manifests/init.pp#L20-L22 it does suggest that this is a pair. However, this commit isn't very clear in **why**:

https://github.com/puppetlabs/puppetlabs-satellite_pe_tools/commit/944683480633108c25d1d384bd5f60f77466fd2e

The fact we can't figure it out with documentation is not a good sign for this module.